xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm

```diff
@@ -33,14 +33,24 @@
   <input type="text" id="xwikidocparentinput" name="parent" value="$!{escapetool.xml("$!docParent")}" size="30" class="suggestDocuments "/>
 </div>
 #if ($editor != 'inline')
-<div id="titleinput" class="form-group">
-  <label for="xwikidoctitleinput">$services.localization.render('core.editors.content.titleField.label')</label>
-  <input type="text" id="xwikidoctitleinput" name="title" value="$escapetool.xml("$!docTitle")" class="#if($xwiki.getXWikiPreference('xwiki.title.mandatory') == 1)required#end"/>
-</div>
+<dl id="titleinput" class="form-group">
+  <dt>
+    <label for="xwikidoctitleinput" value="$escapetool.xml("$!docTitle")">$services.localization.render('core.editors.content.titleField.label')</label>
+  </dt>
+  <dd>
+    <input type="text" id="xwikidoctitleinput" name="title" value="$escapetool.xml("$!docTitle")" 
+      class="#if($xwiki.getXWikiPreference('xwiki.title.mandatory') == 1)required#end"/>
+  </dd>
+</dl>
 #end
 
 #if($editor == 'wiki')
-<div id="contentMeta">
-  <label for="content">$services.localization.render('core.editors.content.contentField.label')</label>
-</div>
+<dl id="contentMeta">
+  <dt>
+    <label for="content" value="$escapetool.xml("$!docTitle")">$services.localization.render('core.editors.content.contentField.label')</label>
+  </dt>
+## Content area has an empty <dd> because the content text area is far below and couldn't be moved here,
+## and the HTML needs to remain valid.
```

Comment:
The comment is good, but needs to be indented (because it applies to the ``dd``, which is indented).

Reply:
Ok